docs/_static/niscope_usage.inc

```diff
@@ -21,41 +21,8 @@ The following is a basic example of using the **niscope** module to open a sessi
         # Find all record number 3
         rec3 = [wfm for wfm in waveforms if wfm.record == 3]
 
-The waveform returned from `fetch <niscope/class.html#fetch>`_ is a flat list of Python objects
-
-    - Attributes:
-
-        -  **relative_initial_x** (float) the time (in seconds) from the trigger to the first sample in the fetched waveform
-        -  **absolute_initial_x** (float) timestamp (in seconds) of the first fetched sample. This timestamp is comparable between records and acquisitions; devices that do not support this parameter use 0 for this output.
-        -  **x_increment** (float) the time between points in the acquired waveform in seconds
-        -  **channel** (str) channel name this waveform was acquired from
-        -  **record** (int) record number of this waveform
-        -  **gain** (float) the gain factor of the given channel; useful for scaling binary data with the following formula:
-
-                voltage = binary data * gain factor + offset
-
-        -  **offset** (float) the offset factor of the given channel; useful for scaling binary data with the following formula:
-
-                voltage = binary data * gain factor + offset
-
-        - **samples** (array of float) floating point array of samples. Length will be of the actual samples acquired
-
-    - Such that all record 0 waveforms are first. For example, with a channel list of 0,1, you would have the following index values:
```

Comment:
The part about record and channel order isn't exactly duplicated information. We don't really document that concept well in our API documentation. Should we keep that bit or perhaps link to some our online fundamental concept documentation?

Reply:
The API documentation doesn't explain how it's determined what order waveforms are returned in. Although we do stick the record number and channel name in the waveform_info, it might be nice for users to know how it works.

Reply:
I guess it's fine cutting it out, for now. Information like that doesn't really belong here, anyway.